nautobot-app/{{ cookiecutter.project_slug }}/{{ cookiecutter.app_name }}/tests/test_basic.py

```diff
@@ -15,8 +17,9 @@ def test_version(self):
         with open(f"{parent_path}/docs/requirements.txt", "r", encoding="utf-8") as file:
             requirements = [line for line in file.read().splitlines() if (len(line) > 0 and not line.startswith("#"))]
         for pkg in requirements:
-            if len(pkg.split("==")) == 2:
-                pkg, version = pkg.split("==")
+            package_name = pkg
+            if len(pkg.split("==")) == 2:  # noqa: PLR2004
+                package_name, version = pkg.split("==")
```

Comment:
Not sure I understand this change - `package_name` isn't used anywhere that I can see, and the below assertion is looking for `pkg` not for `package_name`?

Reply:
cc.: @Kircheneer

Reply:
Yeah, I see I made a mistake here, I meant to use it as the dictionary key further down. The linter was complaining because we  had overwritten the for loop variable and I think this is a valid enough concern.